cmd/simulator/config/flags.go

```diff
@@ -80,12 +106,12 @@ func BuildFlagSet() *pflag.FlagSet {
 func addSimulatorFlags(fs *pflag.FlagSet) {
 	fs.Bool(VersionKey, false, "Print the version and exit.")
 	fs.String(ConfigFilePathKey, "", "Specify the config path to use to load a YAML config for the simulator")
-	fs.StringSlice(EndpointsKey, []string{"ws://127.0.0.1:9650/ext/bc/C/ws"}, "Specify a comma separated list of RPC Websocket Endpoints")
-	fs.Int64(MaxFeeCapKey, 50, "Specify the maximum fee cap to use for transactions denominated in GWei")
-	fs.Int64(MaxTipCapKey, 1, "Specify the max tip cap for transactions denominated in GWei")
-	fs.Uint64(TxsPerWorkerKey, 100, "Specify the number of transactions to create per worker.")
-	fs.Int(WorkersKey, 1, "Specify the number of workers to create for the simulator.")
+	fs.StringSlice(EndpointsKey, []string{"ws://127.0.0.1:9650/ext/bc/C/ws"}, "Specify a comma separated list of RPC Websocket Endpoints (minimum of 1 endpoint)")
+	fs.Int64(MaxFeeCapKey, 50, "Specify the maximum fee cap to use for transactions denominated in GWei (must be > 0)")
+	fs.Int64(MaxTipCapKey, 1, "Specify the max tip cap for transactions denominated in GWei (must be >= 0)")
+	fs.Uint64(TxsPerWorkerKey, 100, "Specify the number of transactions to create per worker (must be > 0)")
+	fs.Int(WorkersKey, 1, "Specify the number of workers to create for the simulator (must be > 0).")
 	fs.String(KeyDirKey, ".simulator/keys", "Specify the directory to save private keys in (INSECURE: only use for testing)")
-	fs.Duration(TimeoutKey, 5*time.Minute, "Specify the timeout for the simulator to complete.")
+	fs.Duration(TimeoutKey, 5*time.Minute, "Specify the timeout for the simulator to complete (0 indicates no timeout).")
```

Comment:
nit: should we be consistent with `.` at the end?

Reply:
No `.`! Technically, you should not have a `.` if you aren't forming a complete sentence and often, error messages or logs are only sentence fragments. To `.` or not to `.` is the bane of my existence, and if you ever review any of my code, I'm sure you will see inconsistent use (especially in comments).

Reply:
yes I mean either consistent with having or not having them :)

Reply:
Gonna update this to no `.`